MAVProxy/modules/mavproxy_map/__init__.py

```diff
@@ -89,6 +89,7 @@ def __init__(self, mpstate):
         self.add_menu(MPMenuItem('Set Origin (with height)', 'Set Origin', '# map setorigin '))
         self.add_menu(MPMenuItem('Terrain Check', 'Terrain Check', '# terrain check'))
         self.add_menu(MPMenuItem('Show Position', 'Show Position', 'showPosition'))
+        self.add_menu(MPMenuItem('Print Google Maps Link', 'Print Google Maps Link', 'printGoogleMapsPosition'))
```

Comment:
'Google Maps Link' is better (for narrower screens)

Reply:
Fixed